README-dev.md

```diff
@@ -74,7 +74,9 @@ We'll need to add this account address in two places, once in the `migration-ove
 
 This step requires you to have the helper tool `celotooljs` installed on your path, if you didn't do that while setting up the monorepo add this to your `.bashrc`/`.zshrc`/`.XXXrc` file `alias celotooljs=<YOUR_PATH_TO_MONOREPO>/packages/celotool/bin/celotooljs.sh`.
 
-Run the following command in a new terminal window or tab to start up a blockchain node:
+1. Make sure you have [built](https://github.com/celo-org/celo-blockchain/blob/master/README.md#building-the-source) the `geth` package on the blockchain repo.
+   
```

Comment:
@pedro-clabs I'm adding this bit to the dev readme. I had to look up how to build the source on the first setup. Hope it's ok.